src/components/DateInput.jsx

```diff
@@ -81,13 +87,29 @@ export default class DateInput extends React.Component {
   }
 
   onKeyDown(e) {
```

Comment:
Is there already a dependency on lodash for this library? Otherwise I'd say it might be worth just creating a util function for that to not have to load an entire library